src/mutils/namespace.py

```diff
@@ -40,7 +41,7 @@ def getFromDagPaths(dagPaths):
         namespace = getFromDagPath(dagPath)
         namespaces.append(namespace)
 
-    return list(set(namespaces))
```

Comment:
What's the reason behind changing this line? It was removing duplicate namespaces before, but now it appears to be returning a list containing many of the same namespaces.

Reply:
Set does not preserve the order of the list which is breaking the Namespace matching. Duplicates get resolved in `getFromSelection`.